### ISSUE TITLE
it is a huge job

### ISSUE THREAD

**Issue by docxu233333333333333333** [fw4spl-org/fw4spl-deps#9]
After several times of trying to compile the deps , I find it is a huuuuuuuuge job, and it is unavoidable to meet errors.
I am deeply grateful if the genius product turns into deb(i found [something](https://packages.debian.org/unstable/main/fw4spl)) or something simple.
plz :)
I am waiting for big news!

**Comment by docxu233333333333333333** [comment]
@fbridault    sorry to bother you again

**Comment by fbridault** [comment]
Yes we know, it is a big work to compile all the dependencies. You can meet compile errors, but if you follow the instructions in http://fw4spl.readthedocs.io/en/fw4spl_0.11.0/Installation/index.html you may minimize them.
Currently we are working in two directions to skip this step : 
1. Precompiled binaries. It currently works on Linux and we should be able to deliver tar.gz for common Linux distributions in January. Windows should follow.
2. Distro packaging. We are working with the DebianMed to package fw4spl, thus you won't need to compile anything. It was even available in [Jessie](https://tracker.debian.org/pkg/fw4spl). It was removed recently from unstable but we are working to get it back. Most of the update job is done (it was an old fw4spl version) and we are waiting for the approval of the [ftpmaster](https://ftp-master.debian.org/new.html), first for [camp](https://tracker.debian.org/pkg/camp), and then for fw4spl.

**Comment by d0cxu** [comment]
@fbridault  Yes, I did everything following the docs.
And thank u very much for what you have done and will do :)
